<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Summary</class>
 <widget class="QDialog" name="Summary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>437</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Summary</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Period :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>Today</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>This Week</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Last Week</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>This Month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Last Month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>This Year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Last Year</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777203</height>
        </size>
       </property>
       <property name="text">
        <string>Total :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_dsbTotal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Categories Info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="m_twTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Graphic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>onShowSummaryGraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onShowSummaryGraph()</slot>
 </slots>
</ui>
